src/biokbase/auth.py

```diff
@@ -29,59 +53,43 @@ def validate_token():
         return False
 
 
-def set_environ_token(token):
+def set_environ_token(token: str) -> None:
     """
     Sets a login token in the local environment variable.
     """
     kbase_env.auth_token = token
 
 
-def get_auth_token():
+def get_auth_token() -> Optional[str]:
     """
     Returns the current login token being used, or None if one isn't set.
     """
     return kbase_env.auth_token
 
 
-def get_user_info(token):
```

Comment:
This was poorly named - it's really getting info about the token. The actual `get_user_info` call (with user anonymous id) will be added next.